### ISSUE TITLE
Multiple jsonPatches

### ISSUE THREAD

**Issue by olfway** [roboll/helmfile#1335]
I'm trying to apply two json patches to chart:

```
repositories:
  - name: nginx-stable
    url: https://helm.nginx.com/stable

releases:
  - name: proxy
    namespace: proxy-ingress
    chart: nginx-stable/nginx-ingress
    version: 0.5.2
    values:
      - controller:
          kind: daemonset
    jsonPatches:
      - target:
          group: apps
          version: v1
          kind: DaemonSet
          name: proxy-nginx-ingress
          namespace: proxy-ingress
        patch:
          - op: add
            path: /spec/template/spec/containers/0/args/-
            value: "-global-configuration=$(POD_NAMESPACE)/nginx-ingress"
      - target:
          group: rbac.authorization.k8s.io
          version: v1beta1
          kind: ClusterRole
          name: proxy-nginx-ingress
        patch:
          - op: add
            path: /rules/-
            value:
              apiGroups:
                - k8s.nginx.org
              resources:
                - globalconfigurations
              verbs:
                - list
                - watch
                - get
```

and getting error:

```
I0707 15:38:37.356093   81787 chartify.go:241] using requirements.yaml:
dependencies:
I0707 15:38:37.403696   81787 replace.go:51] options: {false [/var/folders/v2/7pkc_4x952l8rv9rfgmyxkl80000gn/T/values604541087] [] proxy-ingress 0.5.2 true}
I0707 15:38:37.463584   81787 replace.go:107] removing /var/folders/v2/7pkc_4x952l8rv9rfgmyxkl80000gn/T/2267693928580243482/nginx-ingress/templates/controller-configmap.yaml
I0707 15:38:37.463896   81787 replace.go:107] removing /var/folders/v2/7pkc_4x952l8rv9rfgmyxkl80000gn/T/2267693928580243482/nginx-ingress/templates/controller-daemonset.yaml
I0707 15:38:37.464003   81787 replace.go:107] removing /var/folders/v2/7pkc_4x952l8rv9rfgmyxkl80000gn/T/2267693928580243482/nginx-ingress/templates/controller-deployment.yaml
I0707 15:38:37.464094   81787 replace.go:107] removing /var/folders/v2/7pkc_4x952l8rv9rfgmyxkl80000gn/T/2267693928580243482/nginx-ingress/templates/controller-leader-election-configmap.yaml
I0707 15:38:37.464192   81787 replace.go:107] removing /var/folders/v2/7pkc_4x952l8rv9rfgmyxkl80000gn/T/2267693928580243482/nginx-ingress/templates/controller-secret.yaml
I0707 15:38:37.464255   81787 replace.go:107] removing /var/folders/v2/7pkc_4x952l8rv9rfgmyxkl80000gn/T/2267693928580243482/nginx-ingress/templates/controller-service.yaml
I0707 15:38:37.464323   81787 replace.go:107] removing /var/folders/v2/7pkc_4x952l8rv9rfgmyxkl80000gn/T/2267693928580243482/nginx-ingress/templates/controller-serviceaccount.yaml
I0707 15:38:37.464390   81787 replace.go:107] removing /var/folders/v2/7pkc_4x952l8rv9rfgmyxkl80000gn/T/2267693928580243482/nginx-ingress/templates/controller-ts-definition.yaml
I0707 15:38:37.464479   81787 replace.go:107] removing /var/folders/v2/7pkc_4x952l8rv9rfgmyxkl80000gn/T/2267693928580243482/nginx-ingress/templates/controller-vs-definition.yaml
I0707 15:38:37.464615   81787 replace.go:107] removing /var/folders/v2/7pkc_4x952l8rv9rfgmyxkl80000gn/T/2267693928580243482/nginx-ingress/templates/controller-vsr-definition.yaml
I0707 15:38:37.464753   81787 replace.go:107] removing /var/folders/v2/7pkc_4x952l8rv9rfgmyxkl80000gn/T/2267693928580243482/nginx-ingress/templates/controller-wildcard-secret.yaml
I0707 15:38:37.464824   81787 replace.go:107] removing /var/folders/v2/7pkc_4x952l8rv9rfgmyxkl80000gn/T/2267693928580243482/nginx-ingress/templates/rbac.yaml
I0707 15:38:37.465040   81787 patch.go:37] patching files: [/var/folders/v2/7pkc_4x952l8rv9rfgmyxkl80000gn/T/2267693928580243482/nginx-ingress/helmx.1.rendered/nginx-ingress/templates/all.yaml]
I0707 15:38:37.465498   81787 patch.go:101] jsonpatches/patch.0.yaml:
- op: add
  path: /spec/template/spec/containers/0/args/-
  value: -global-configuration=$(POD_NAMESPACE)/nginx-ingress
in ./helmfile.yaml: [mkdir /var/folders/v2/7pkc_4x952l8rv9rfgmyxkl80000gn/T/2267693928580243482/nginx-ingress/jsonpatches: file exists]
```

I'm using helmfile version v0.120.0

**Comment by dbayub** [comment]
I've experienced the same problem. It should not fail if the directory already exists, equivalent to using the "-p" option to mkdir.

**Comment by mumoshu** [comment]
Thanks for reporting! This is going to be fixed via #1359
